openhexa/sdk/workspaces/connection.py

```diff
@@ -7,6 +7,9 @@ class DHIS2Connection:
     username: str
     password: str
 
+    def __repr__(self):
+        return f"DHIS2Connection(url='{self.url}', username='{self.username}', password='*********')"
```

Comment:
Just don't add the password to the repr, it's easier and do not try to convey a message (that can be false)

Reply:
So 
```python
def __repr__(self):
   return f"DHIS2Connection(url='{self.url}', username='{self.username}')"
```
?